nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/grok/GrokReader.java

```diff
@@ -74,6 +74,8 @@
         "The line of text that does not match the Grok Expression will be appended to the last field of the prior message.");
     static final AllowableValue SKIP_LINE = new AllowableValue("skip-line", "Skip Line",
         "The line of text that does not match the Grok Expression will be skipped.");
+    static final AllowableValue THROW_ERROR = new AllowableValue("throw-error", "Error",
+            "The processing of the flow file containing the line of text will throw an error and will be routed to the approriate relationship.");
```

Comment:
Nitpick, "throw an error" might be too dev-centric, perhaps "issue an error"? Not necessary just a suggestion. Also there is a typo in "appropriate", and the PR won't currently merge cleanly, do you mind doing a rebase? Please and thanks!

Reply:
Done, thanks @mattyb149 !